components/compass-runtime-agent/internal/compass/director/client.go

```diff
@@ -12,8 +12,8 @@ import (
 const (
 	TenantHeader = "Tenant"
 
-	eventsURLLabelKey  = "runtime/event_service_url"
-	consoleURLLabelKey = "runtime/console_url"
+	eventsURLLabelKey  = "runtime_eventServiceUrl"
+	consoleURLLabelKey = "runtime_consoleUrl"
```

Comment:
Question: is Kyma Environment Broker adjusted to this change? Compass Runtime Agent labels a runtime and KEB later reads label's value.

Reply:
Yes, it is adjusted here https://github.com/kyma-incubator/compass/pull/984